src/frequenz/sdk/data_ingestion/resampling/component_metrics_resampling_actor.py

```diff
@@ -0,0 +1,240 @@
+"""
+ComponentMetricsResamplingActor used to subscribe for resampled component metrics.
```

Comment:
should this move to `src/frequenz/sdk/actor/resampling`?

Reply:
> ComponentMetricsResamplingActor

Its not resampling multiple metrics at the same it, or is it? Otherwise we might want to kick out the `s` after Metric ;-)

Reply:
It does resample multiple metrics - all metric requests will go to this actor,  but I'm still ok with removing the `s` and calling it `ComponentMetricResamplingActor` :-)

Reply:
> should this move to `src/frequenz/sdk/actor/resampling`?

Yeah, I will do so in future PRs, I didn't want to keep blocking the PR for small details, specially when the directory structure is not very well defined yet (I think while we add more stuff we would have more clarity on a structure that makes sense).

IMHO there are 2 levels here. The actor should definitely go to `src/frequenz/sdk/actor/resampling` but the classes should probably go to `src/frequenz/sdk/data_pipeline/resampling`, as they are meant to be reused by other actors. IMHO only actors (and message classes to interact with those actors) should be imported from `sdk/actor`.

About `s` or not `s`, for me it makes more sense with the `s` because it is an actor that resamples component metrics, but I won't fight it if anyone sends a PR to change it :grimacing: ...

Reply:
I will do this as part of:

* #56

Reply:
I was thinking about `timeseries`, which is the name that we were using internally. At first I dropped it because we were only resampling component metrics, but actually the classes don't need to know about component metrics, so they can be still called `TimeseriesResampler`. Another option would be to just put it in `sdk.resampling`. Again, I think we'll get more clarity as the SDK grows, my impression right now is that any name we pick now has very high chances of being changed in the future :D

Reply:
So `Sample` would also make sense to live in `sdk.timeseries`, and the current requests for the `DataSourcingActor` should be moved to the actor module IMHO if we chose this path.

Reply:
> So Sample would also make sense to live in sdk.timeseries, and the current requests for the DataSourcingActor should be moved to the actor module IMHO if we chose this path.

agreed.